test/unit/integrations/bit_pay/bit_pay_notification_test.rb

```diff
@@ -25,42 +25,47 @@ def test_compositions
   end
 
   def test_successful_acknowledgement_with_v1_api_token
-    stub_request(:get, "#{BitPay.invoicing_url(@token_v1)}/#{@invoice_id}").
-      to_return(status: 200, body: http_raw_data)
+    stub_request(:get, "#{BitPay::API_V2_URL}/#{@invoice_id}")
+      .with(basic_auth: [@token_v1, ''])
```

Comment:
this is not needed (.with(basic_auth: [@token_v1, ''])), also you could remove one of the tests since there's no difference between token versions in this case